### PR TITLE
task 291 & 295 : semeval 2020 task4

### PR DESCRIPTION
**Pull request by amirrezamirzaei**
task 291 & 295 using [semeval 2020 task 4](https://github.com/wangcunxiang/SemEval2020-Task4-Commonsense-Validation-and-Explanation)

### CODE REVIEW

**Comment by Palipoor** [comment]
I am not sure, but maybe "Classification" would be a better fit for task 281. 
Also, it may be better if the sentences in task 281 are not separated by comma, and are separated by titles like: sentence 1 : ... sentence 2: ... . In this case, the answers can be "sentence 1" and "sentence 2".
In your examples for task 281, please add more explanation, i.e. why an statement does or does not make sense.

**Review by Palipoor — COMMENTED** [review]
I am not sure, but maybe "Classification" would be a better fit for task 281.
Also, it may be better if the sentences in task 281 are not separated by comma, and are separated by titles like: sentence 1 : ... sentence 2: ... . In this case, the answers can be "sentence 1" and "sentence 2".
In your examples for task 281, please add more explanation, i.e. why an statement does or does not make sense.

**Review by Palipoor — COMMENTED** [review]
In task 295:
` that is against commonsense.you are`: Please capitalize the start of the sentences, and put a space after comma, period, etc. I think the definition is not very well written.
Also, like the other task, please explain more in the examples, and add negative examples that have valid, but not correct outputs.

**Comment by amirrezamirzaei** [comment]
thanks for the suggestions. I think answer generation is a better fit for task 291, the question being "which sentence doesn't make sense". using classification would imply we are classifying all the statements but we are just comparing two of them for each instance.

also Im not very sure about adding more explanation for the examples because explaining why one statement doesn't make sense and the other does is kinda obvious. the examples in this case are more for showing how the output should be generated.

**Comment by swarooprm** [comment]
Looks good!

File: tasks/README.md [hunk 1]
```diff
@@ -232,6 +232,8 @@ Name | Summary | Category
 `task274_overruling_legal_classification`	| Given a sentence, classify it into overruling or non-overruling| Legal , Classification
 `task275_enhanced_wsc_paraphrase_generation` | Given a sentence and an aspect, paraphrase the sentence changing that aspect | Text Modification
 `task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
+`task291_semeval_2020_task4_Commonsense_Validation` | Given two statement, choose the one that makes less sense. | Answer Generation
+`task295_semeval_2020_task4_Commonsense_Reasoning` | Given an statement against commonsense and 3 reasons, choose the best reason explaining why the statement is against commonsense  | Answer Generation
```
Comment (danyaljj) [inline tasks/README.md]: **a** statement

File: tasks/README.md [hunk 2]
```diff
@@ -232,6 +232,8 @@ Name | Summary | Category
 `task274_overruling_legal_classification`	| Given a sentence, classify it into overruling or non-overruling| Legal , Classification
 `task275_enhanced_wsc_paraphrase_generation` | Given a sentence and an aspect, paraphrase the sentence changing that aspect | Text Modification
 `task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
+`task291_semeval_2020_task4_Commonsense_Validation` | Given two statement, choose the one that makes less sense. | Answer Generation
```
Comment (danyaljj) [inline tasks/README.md]: two statement**s** (plural)

**Comment by danyaljj** [comment]
Thanks! 
Indicated several typos.  Let's do a careful grammar check before merging the PR. Might be helpful to try https://app.grammarly.com/ for a quick check.

**Pull request merged**